src/store/reducers/posts.js

```diff
@@ -0,0 +1,15 @@
+const INITIAL_POST_STATE = {
+  posts: [],
+};
+
+export default (state = INITIAL_POST_STATE, action) => {
+  switch (action.type) {
+    case "GET_ALL_POSTS":
```

Comment:
Related to comment above for action names

Reply:
done